tests/test_movie_json_generator_with_diff_n.py

```diff
@@ -0,0 +1,234 @@
+# a re-implementation of test_movie_json_generator.py with a
+# different number of frames per movie
```

Comment:
this needs to be parameterized using `pytest.mark.parameterize` instead of adding an entire new test file